TypeCobol/Compiler/Nodes/ParametersProfileNode.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace TypeCobol.Compiler.Nodes
 {
-    public class ParametersProfileNode : GenericNode<ParametersProfile>, ParameterList
+    public class ParametersProfileNode : GenericNode<ParametersProfile>, ParameterList, IEquatable<ParametersProfileNode>
```

Comment:
Equals and GetHashCode could be forwarded to ParametersProfile to avoid duplicate code.

Reply:
done.
https://github.com/TypeCobolTeam/TypeCobol/pull/1600/commits/8bb22a0bed7c51638872bc38edfbcb12c9e2ab34